Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -192,9 +305,13 @@ def search_slack_users(users: Union[list, str]) -> list:
     for user in users:
         slack_user = get_user_by_name(user)
         if not slack_user:
+            err_str = f'User {user} not found in Slack'
+            if SAFE_MODE:
+                err_str += ' and Safe Mode is enabled. If this command worked previously for you, please try disabling' \
+                           ' Safe Mode from the instance configuration or use the users email instead.'
```

Comment:
please direct to the limitations and warning about disabling safe mode in the readme.

Reply:
and maybe put this error message in constant as you are using it 4 times.